change/react-native-windows-37128f50-e855-4562-9019-16b5480fceb4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "CrashManager",
```

Comment:
>CrashManager [](http://example.com/codeflow?start=14&length=12)

nit: it would be better to expand it.

Reply:
Agreed, - this string shows up in the changelog